roles/livekit/defaults/main.yml

```diff
@@ -75,11 +95,35 @@ livekit_container_combined_volumes: >-
   {{ livekit_container_preset_volumes + livekit_container_volumes }}
 livekit_container_network_mode: "host"
 livekit_jwt_service_container_name: "jwt-service"
-livekit_jwt_service_container_image_reference: "docker-oss.nexus.famedly.de/lk-jwt-service:{{ livekit_jwt_service_version }}"
-livekit_jwt_service_container_env:
+livekit_jwt_service_container_image_reference: >-
+  {{
+    livekit_jwt_service_container_image_repository
+    + ':'
+    + livekit_jwt_service_container_image_tag | default('v' + livekit_jwt_service_version)
+  }}
+livekit_jwt_service_container_image_repository: >-
+  {{
+    (
+      container_registries[livekit_jwt_service_container_image_registry]
+      | default(livekit_jwt_service_container_image_registry)
+    )
+    + '/'
+    + livekit_jwt_service_container_image_namespace | default('')
+    + livekit_jwt_service_container_image_name
+  }}
+livekit_jwt_service_container_image_registry: "docker-oss.nexus.famedly.de"
+livekit_jwt_service_container_image_name: "lk-jwt-service"
+livekit_jwt_service_container_env_base:
   LIVEKIT_KEY: "secret"
   LIVEKIT_SECRET: "{{ livekit_secret_key }}"
   LIVEKIT_URL: "wss://{{ livekit_domain }}"
-  HS_ALLOWLIST: "*.famedly.de, *.famedly.care"
+livekit_jwt_service_homeserver_allowlist: ""
```

Comment:
I'd prefer if this was not a string, but an array. We can do the conversion into a string with `| join(', ')` afterwards

Reply:
Should be fine now.